src/Digdir.Domain.Dialogporten.Application/Common/MappingUtils.cs

```diff
@@ -0,0 +1,28 @@
+using System.Security.Cryptography;
+
+namespace Digdir.Domain.Dialogporten.Application.Common;
+
+internal static class MappingUtils
+{
+    internal static byte[] GetHashSalt(int size = 16) => RandomNumberGenerator.GetBytes(size);
+
+    internal static string? HashPid(string? personIdentifier, byte[] salt)
+    {
+        if (string.IsNullOrWhiteSpace(personIdentifier))
+        {
+            return null;
+        }
+
+        const int keySize = 32;
+        const int iterations = 5_000;
```

Comment:
Tipper dette ikke er spesielt dyrt, men vi må passe på at vi ikke betaler flere millisekunder her. Lurt å ha salt static per instans av applikasjonen.

Reply:
Jeg lurer på om vi kan bytte ut med noe non-cryptographic som er mye kjappere, om vi allikevel skal sende bare de ~5 første karakterene?

Reply:
Leste koden feil, ser det er ny salt per request. Men det er greit nok, og da tenker jeg det ihvertfall ikke er noe behov for å stretche den hashen noe, slik at en ren SHA256 vil være plenty, altså bare

```
        var identifierBytes = Encoding.UTF8.GetBytes(personIdentifier);
        Span<byte> buffer = stackalloc byte[identifierBytes.Length + salt.Length];
        identifierBytes.CopyTo(buffer);
        salt.CopyTo(buffer[identifierBytes.Length..]);

        var hashBytes = SHA256.HashData(buffer);

        return BitConverter.ToString(hashBytes,0, 5).Replace("-", "").ToLowerInvariant();
```